lib/universal/git-hooks.ts

```diff
@@ -0,0 +1,108 @@
+
+import { parse } from "https://deno.land/std/flags/mod.ts";
+import { Command } from "https://deno.land/x/cliffy/command/mod.ts";
+
+async function initHooks() {
+  const gitHooksDir = '.githooks';
+  await Deno.mkdir(gitHooksDir, { recursive: true });
+
+  const hooks = {
+    'pre-commit': preCommit,
+    'pre-push': prePush,
+    'prepare-commit-message': prepareCommitMessage,
+  };
+
+  for (const [name, handler] of Object.entries(hooks)) {
+    const script = `#!/usr/bin/env -S deno run --allow-read --allow-write --allow-env
+import { ${name} } from "https://raw.githubusercontent.com/netspective-labs/sql-aide/vX.Y.Z/lib/universal/git-hooks.ts";
+await ${name}();`;
+
+    await Deno.writeTextFile(`${gitHooksDir}/${name}`, script, { mode: 0o755 });
```

Comment:
use Deno stdlib `path` module and `path.join()` not string concatenation for filename.